### DISCUSSION TITLE
RSocket current status？

### DISCUSSION THREAD

**Discussion by lizhongyue248** [rsocket/rsocket-java#1109] [Q&A]
The current RSocket libraries have not been updated for a long time, and the official Twitter also stopped updating, the current status of the project is what?

I wanted to use it on my project, but it didn't seem to be active, and it wasn't being updated or maintained, which worried me.

**Comment by GarnicaJR** [comment]
It has been stuck since a long time ago, I am using in my legacy project to replace a custom TCP implementation

Specifically telemetry area to collect data from POS sales devices
